File: example/fundamentals/modules/7-root-module/test/src/foo/FooTest.java
package foo;

import static com.google.common.html.HtmlEscapers.htmlEscaper;
import static org.junit.Assert.assertEquals;

import org.junit.Test;

public class FooTest {
  @Test
  public void testSimple() {
    assertEquals(Foo.generateHtml("hello"), "<h1>hello</h1>");
  }

  @Test
  public void testEscaping() {
    assertEquals(Foo.generateHtml("<hello>"), "<h1>" + htmlEscaper().escape("<hello>") + "</h1>");
  }
}

File: example/kotlinlib/basic/1-simple/test/src/foo/FooTest.kt
package foo

import io.kotest.core.spec.style.FunSpec
import io.kotest.matchers.shouldBe

class FooTest :
    FunSpec({
        test("testSimple") {
            generateHtml("hello") shouldBe "<h1>hello</h1>"
        }

        test("testEscaping") {
            generateHtml("<hello>") shouldBe "<h1>&lt;hello&gt;</h1>"
        }
    })
